### ISSUE TITLE
File Not Found Error

### ISSUE THREAD

**Issue by ChuyiZhong** [val-iisc/lsc-cnn#18]
FileNotFoundError: [Errno 2] No such file or directory: './models/train2/snapshots/losses.pkl'
How should we place the downloaded folder models, I have placed it anywhere possible buy still get this error.
Any help is appreciated.

**Comment by pvskand** [comment]
You'd have to put the `models` folder inside `lsc-cnn` as mentioned here (https://github.com/val-iisc/lsc-cnn#usage).

**Comment by ChuyiZhong** [comment]
> You'd have to put the `models` folder inside `lsc-cnn` as mentioned here (https://github.com/val-iisc/lsc-cnn#usage).

Hi, thank you for your quick reply. 
When I cloned the project I found another folder named lsc-cnn in the project folder which is also called lsc-cnn, so I am a little confused. 
So should I just put the models folder in the project lsc-cnn along with network.py main.py or outside the folder along with dataset folder? Hope my expression is not confusing.
Thanks in advance.
